src/Aspire.Hosting.Azure/Provisioning/Provisioners/IBicepProvisioner.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Configuration;
+
+namespace Aspire.Hosting.Azure.Provisioning;
+
+/// <summary>
+/// Provides functionality for provisioning Azure Bicep resources.
+/// </summary>
+internal interface IBicepProvisioner
```

Comment:
This interface exists to support testability. We need an IBicepProvisioner implementation that no-ops in tests.